Packs/AWS-Enrichment-Remediation/Playbooks/playbook-Cloud_Credentials_Rotation_-_AWS.yml

```diff
@@ -619,25 +619,17 @@ tasks:
       policyDocument:
         simple: |-
           {
-              "Version": "2012-10-17",
-              "Statement": [
-                  {
-                      "Effect": "Deny",
-                      "Action": [
-                          "*"
-                      ],
-                      "Resource": [
-                          "*"
-                      ],
-                      "Condition": {
-                          "DateLessThan": {
-                              "aws:TokenIssueTime": "[policy creation time]"
-                          }
-                  }
-              ]
+            "Version": "2012-10-17",
```

Comment:
should we keep this hard-coded version date?

Reply:
Yes, it's a variable.